backend/wait_for_db.py

```
@@ -1,20 +1,22 @@
             print("Waiting for database...")
             db_conn = MySQLdb.connect(
-                host='db',  # The service name defined in your docker-compose.yml
-                user='mhpcoder',
-                password='justkidding12!@',
-                database='learnathon',
-                port='3306'
+                host=os.getenv('DATABASE_HOST'),  # The service name defined in your docker-compose.yml
+                user=os.getenv('DATABASE_USER'),
+                password=os.getenv('DATABASE_PASSWORD'),
+                database=os.getenv('DATABASE_NAME'),
+                port=os.getenv('DATABASE_PORT')
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)